src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+            }
+          }
+        }
+      }
+    },
+    "/requests/{requestUuid}/expenses": {
+      "get": {
+        "tags": [
+          "Expense Resource"
+        ],
+        "summary": "Get expected expenses attached to a request",
+        "operationId": "listUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "The unique identifier of the Request",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user viewing the expected expenses attached to a Request",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ResourceOfExpense"
+              }
+            }
+          },
+          "400": {
+            "description": "The request is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Request is not authenticated"
+          },
+          "403": {
+            "description": "Principal is not allowed to perform operation"
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Expense Resource"
+        ],
+        "summary": "Create a new expected expense",
+        "operationId": "createUsingPOST_2",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "The unique identifier of the Request to which the Expense is attached",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "expense",
+            "description": "expense",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Expense"
+            }
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user performing the expense creation",
```

Comment:
The unique identifier of the user performing the Expense creation

Reply:
The unique identifier of the user performing the expected expense creation